angular-ngrx-scss/src/app/state/profile/profile.reducer.ts

```diff
@@ -0,0 +1,18 @@
+import { Action, createReducer, on } from '@ngrx/store';
+import { fetchProfileSuccess } from './profile.actions';
+import { ProfileState } from './profile.state';
+
+export const initialState: ProfileState = {};
```

Comment:
I feel like this should have the default values for the values we expect here, instead of being an empty object. But perhaps it's not needed? @ktrz @morgnism would appreciate your thoughts here, not sure if there's a preferred best practice for that or not.

Reply:
I don't know what the best practice here. I made the values nullable so this spot would not need to be refactored to add empty values if we added new fields.